desktop/core/src/desktop/api2.py

```diff
@@ -813,6 +814,44 @@ def export_documents(request):
     return make_response(f.getvalue(), 'json', filename)
 
 
+def topologicalSort(docs):
```

Comment:
Are we assuming that our data is acyclic or does the code take care of it? Is there so standard library way of getting this done. I am just trying to explore the possibility of using some standard library for the sake of maintainability. Otherwise, this is good. Please add more comments in the code.

Reply:
Nice catch, No code is not taking care of it and do we need to look into it @amitsrivastava ?
I am not sure do we any standard library for this.
Yes, sure I will add more comments
Thanks